parachain/pallets/inbound-queue/Cargo.toml

```diff
@@ -33,12 +33,11 @@ snowbridge-ethereum = { path = "../../primitives/ethereum", default-features = f
 snowbridge-router-primitives = { path = "../../primitives/router", default-features = false }
 ethabi = { git = "https://github.com/Snowfork/ethabi-decode.git", package = "ethabi-decode", branch = "master", default-features = false }
 
-snowbridge-beacon-primitives = { path = "../../primitives/beacon", optional = true }
+snowbridge-beacon-primitives = { path = "../../primitives/beacon", default-features = false, optional = true }
 
 [dev-dependencies]
 frame-benchmarking = { git = "https://github.com/paritytech/substrate.git", branch = "master" }
 sp-keyring = { git = "https://github.com/paritytech/substrate.git", branch = "master" }
-snowbridge-beacon-primitives = { path = "../../primitives/beacon" }
```

Comment:
Seems it's still required for the unit test or the CI will fail. Or we can just add it into the benchmark feature as in 
https://github.com/Snowfork/snowbridge/blob/b3c4288780ac622c68a3110468691c27eac21ef0/parachain/pallets/inbound-queue/Cargo.toml#L74